monitoring/dashboard.json

```diff
@@ -0,0 +1,627 @@
+{
+    "__inputs": [
+        {
+            "name": "DS_PROMETHEUS",
+            "label": "Prometheus",
+            "description": "",
+            "type": "datasource",
+            "pluginId": "prometheus",
+            "pluginName": "Prometheus"
+        },
+        {
+            "name": "namespace",
+            "type": "constant",
+            "label": "namespace",
+            "description": "Namespace associated with the Zenko instance",
+            "value": "zenko"
+        },
+        {
+            "name": "job",
+            "type": "constant",
+            "label": "job",
+            "description": "Name of the Redis job, used to filter only the Redis instances.",
+            "value": "artesca-data-connector-s3api-metrics"
+        }
+    ],
+    "annotations": {
+        "list": [
+            {
+                "builtIn": 1,
+                "datasource": "-- Grafana --",
+                "enable": true,
+                "hide": true,
+                "iconColor": "rgba(0, 211, 255, 1)",
+                "name": "Annotations & Alerts",
+                "type": "dashboard"
+            }
+        ]
+    },
+    "editable": true,
+    "gnetId": null,
+    "graphTooltip": 0,
+    "id": 33,
+    "links": [],
+    "panels": [
+        {
+            "collapsed": false,
+            "datasource": null,
+            "gridPos": {
+                "h": 1,
+                "w": 24,
+                "x": 0,
+                "y": 0
+            },
+            "id": 2,
+            "panels": [],
+            "title": "Response codes",
+            "type": "row"
+        },
+        {
+            "datasource": null,
```

Comment:
should not be NULL : this is there is a 'prometheus' datasource variable

Reply:
this is odd I have `"datasource": null` everywhere :open_mouth: 
but It works fine when I created the dashboard

Reply:
`datasource: null` will always use the "default" datasource, which is actually prometheus...